libcloud/compute/drivers/ec2.py

```diff
@@ -1783,12 +1783,28 @@ def list_locations(self):
             )
         return locations
 
-    def list_volumes(self, node=None):
+    def list_volumes(self, node=None, filters=None):
```

Comment:
Since it's an extension argument please rename it to ``ex_filters``.

Reply:
Fixed.